### PR TITLE
Update json file example

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes

- https://docs.docker.com/ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster/#compatibility-with-daemonjson
- Update the `Compatibility with daemon.json`  example

### Related issues (optional)
https://github.com/docker/docker.github.io/issues/9114

### CODE REVIEW

**Comment by traci-morrison** [comment]
@usha-mandya @KoryKessel-Docker PTAL. I verified that the information is correct via Microsoft's KB.

File: ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster.md [hunk 1]
```diff
@@ -75,9 +77,9 @@ $script = [ScriptBlock]::Create((docker run --rm {{ page.ucp_org }}/ucp-agent-wi
 Invoke-Command $script
 ```
 
-> Docker daemon restart
+> Note 
 >
-> When you run `windows-script`, the Docker service is unavailable temporarily.
+> When restarting Docker daemon, note that when running `windows-script`, the Docker service is unavailable temporarily.
```
Comment (usha-mandya) [inline ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster.md]: Can we rephrase this sentence? Not sure which of these condition is correct?

File: ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster.md [hunk 2]
```diff
@@ -99,28 +101,26 @@ to the corresponding files in `C:\ProgramData\docker\daemoncerts`:
...
+To join the cluster using the `docker swarm join` command provided by the Docker UCP web interface and CLI:
 
 1.  Log in to the Docker UCP web interface with an administrator account.
 2.  Navigate to the **Nodes** page.
 3.  Click **Add Node** to add a new node.
 4.  In the **Node Type** section, click **Windows**.
-5.  In the **Step 2** section, click the checkbox for
-    "I'm ready to join my windows node."
-6.  Check the **Use a custom listen address** option to specify the address
+5.  In the **Step 2** section, select the check box for "I have followed the instructions and I'm ready to join my windows node."
```
Comment (usha-mandya) [inline ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster.md]: Should this be Windows with uppercase W?

**Comment by usha-mandya** [comment]
@traci-morrison Looks good with some minor suggestions noted above.

**Comment by traci-morrison** [comment]
> @traci-morrison Looks good with some minor suggestions noted above.

Thanks @usha-mandya. I made your suggested changes. Can you review again?

**Pull request merged**